pkg/volume/validation/pv_validation.go

```diff
@@ -56,3 +60,16 @@ func checkMountOption(pv *api.PersistentVolume) field.ErrorList {
 	}
 	return allErrs
 }
+
+// ValidateHostPath will make sure targetPath:
+// 1. does not have any element which is ".."
+func ValidateHostPath(targetPath string) error {
```

Comment:
Name this more specifically for what it does (ValidateFilePathNoBacksteps or something)